src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/configuration/configuration-form/configuration-form.types.ts

```diff
@@ -0,0 +1,120 @@
+import { Validators } from '@angular/forms';
+import { CdValidators } from '../../../../shared/forms/cd-validators';
+import { ConfigFormModel } from './configuration-form.model';
+
+import * as _ from 'lodash';
+
+export class ConfigOptionTypes {
+  // TODO: I18N
+  private static knownTypes: Array<any> = [
+    {
+      name: 'uint64_t',
+      inputType: 'number',
+      humanReadable: 'Positive integer value',
+      defaultMin: 0,
+      patternHelpText: 'The entered value needs to be a positive number.',
+      isNumberType: true,
+      allowsNegative: false
+    },
+    {
+      name: 'int64_t',
+      inputType: 'number',
+      humanReadable: 'Integer value',
+      patternHelpText: 'The entered value needs to be a number.',
+      isNumberType: true,
+      allowsNegative: true
+    },
+    {
+      name: 'size_t',
+      inputType: 'number',
+      humanReadable: 'Positive integer value (size)',
+      defaultMin: 0,
+      patternHelpText: 'The entered value needs to be a positive number.',
+      isNumberType: true,
+      allowsNegative: false
+    },
+    {
+      name: 'secs',
+      inputType: 'number',
+      humanReadable: 'Positive integer value (secs)',
+      defaultMin: 1,
+      patternHelpText: 'The entered value needs to be a positive number.',
+      isNumberType: true,
+      allowsNegative: false
+    },
+    {
+      name: 'double',
+      inputType: 'number',
+      humanReadable: 'Decimal value',
+      patternHelpText: 'The entered value needs to be a number or decimal.',
+      isNumberType: true,
+      allowsNegative: true
+    },
+    { name: 'std::string', inputType: 'text', humanReadable: 'Text', isNumberType: false },
+    {
+      name: 'entity_addr_t',
+      inputType: 'text',
+      humanReadable: 'IPv4 or IPv6 address',
+      patternHelpText: 'The entered value needs to be a valid IP address.',
+      isNumberType: false
+    },
+    {
+      name: 'uuid_d',
+      inputType: 'text',
+      humanReadable: 'UUID',
+      patternHelpText:
+        'The entered value is not a valid UUID, e.g.: 67dcac9f-2c03-4d6c-b7bd-1210b3a259a8',
+      isNumberType: false
+    },
+    { name: 'bool', inputType: 'checkbox', humanReadable: 'Boolean value', isNumberType: false }
+  ];
+
+  public static getType(type: string): any {
+    const currentType = _.find(this.knownTypes, (t) => {
+      return t.name === type;
+    });
+
+    if (currentType !== null) {
```

Comment:
Nice... seems like the type definition has been changed for the config options.

Reply:
Fix: https://github.com/ceph/ceph/pull/25876

Reply:
It would make sense to fix this line and the corresponding test in #25876 as this caused the type change to be undetected.

Reply:
I would leave it here (merge https://github.com/ceph/ceph/pull/25876 before this one) because the error occurs on master (on the configurations page) already.